examples/java-blob/README.md

```diff
@@ -0,0 +1,215 @@
+# Spring Boot + Azure Storage + AAD Pod Identity
+
+This tutorial is based on [this documentation](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-storage).
+
+This repository contains sample code and instructions demonstrating how you can access Azure Storage using a user-assigned identity with the aid of AAD Pod Identity from a spring boot application.
+
+This sample takes a `POST` request and saves it as a text file in an Azure Blob Storage container.
+
+## Prerequisites
+
+* AKS cluster
+* AAD Pod Identity installed and configured
+* Azure Container Registry
+* Azure Storage
+    * Make sure you have already created a container
+
+## Setup
+
+### Create Manage Identity and Assign Role
+
+In this step, we'll create a new user-assigned identity which will be used to interact with the Azure Storage account.
+
+**Note:** this step can be skipped if you already have a manage identity you'd like to reuse.
+
+1. Setup:
+
+    ```sh
+    CLUSTER_NAME=<YOUR_AKS_CLUSTER_NAME>
+    CLUSTER_RESOURCEGROUP=$(az aks list --query "[?name == '$CLUSTER_NAME'].resourceGroup" -o tsv)
+
+    IDENTITY_NAME=<YOUR_IDENTITY_NAME>
+    IDENTITY_RESOURCE_GROUP=<YOUR_RESOURCE_GROUP>
```

Comment:
For simplicity, we can use the node resource group to store the user-assigned identity:
```suggestion
    IDENTITY_RESOURCE_GROUP=$(az aks show -g $CLUSTER_RESOURCE_GROUP -n $CLUSTER_NAME --query nodeResourceGroup -otsv)
```